src/main/java/com/khutircraftubackend/seller/SellerController.java

```diff
@@ -21,10 +18,9 @@ public class SellerController {
 
     private final SellerService sellerService;
 
-    @PostMapping
-    ResponseEntity<SellerDTO> createSeller(@Valid @RequestBody SellerDTO sellerDTO) {
-        SellerDTO createdSeller = sellerService.saveSeller(sellerDTO);
-        return ResponseEntity.ok(createdSeller);
+    @GetMapping ("/info")
+    public SellerResponse getInfoSeller(Principal principal){
```

Comment:
please rename this method into "getSellerInfo"

Reply:
Rename method to getSellerInfo().